matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/permalinks/PermalinkParser.kt

```diff
@@ -61,20 +64,14 @@ object PermalinkParser {
                 .map { URLDecoder.decode(it, "UTF-8") }
                 .take(2)
 
-        // the element-based domain permalinks (e.g. https://app.element.io/#/user/@chagai95:matrix.org) don't have the
-        // mxid in the first param (like matrix.to does - https://matrix.to/#/@chagai95:matrix.org) but rather in the second after /user/ so /user/mxid
-        var identifier = params.getOrNull(0)
-        if (identifier.equals("user")) {
-            identifier = params.getOrNull(1)
-        }
-
+        val identifier = params.getOrNull(0)
```

Comment:
@chagai95 heads up that this will remove your patch, but I think with the new code it will work. Just wanted you to have a look, since you have changed the value of "PermalinkService.MATRIX_TO_URL_BASE", I think this will actually break something on your side.